### DISCUSSION TITLE
Using MobX with remix for a React project

### DISCUSSION THREAD

**Discussion by arunaugustine** [remix-run/remix#1867] [Q&A] [answered]
Remix as the framework and mobx as the data store? Is it a good idea?

**Comment by hollandThomas** [comment, marked as answer]
In general you don't need any of the "global state managers". Remix has you covered here, unless there's lots of global client-only state. If that's the case, you can totally use MobX or something more lightweight like [zustand](https://github.com/pmndrs/zustand).

For server-state, you can think of your database as your global store

**Reply by artemis-prime** [reply]
Should one follow the same patterns as outlined for nextjs re ssr? 

eg: https://dev.to/ivandotv/mobx-server-side-rendering-with-next-js-4m18
